client/src/App/index.js

```diff
@@ -57,22 +56,25 @@ function App() {
     },
   ];
   return (
-    <AuthProvider>
-      <Router>
+    <Router>
+      <AuthProvider>
         <Switch>
-          {
-          routes.map((route) => (route.public
-            ? <Route key={route.path} exact={route.exact} path={route.path}>{route.children}</Route>
-            : (
-              <ProtectedRoute key={route.path} exact={route.exact} path={route.path}>
-                {route.children}
-
-              </ProtectedRoute>
-            )))
-        }
+          {routes.map((route) => (route.public ? (
```

Comment:
:+1:

Reply:
One little note on this, it doesn't look good, fix your indentation.

Reply:
Done